website/static/js/tests/fangorn.test.js

```diff
@@ -0,0 +1,92 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
+'use strict';
+
+var assert = require('chai').assert;
+var $ = require('jquery');
+// var m = require('mithril');
+// var URI = require('URIjs');
+// var Treebeard = require('treebeard');
+var Fangorn = require('js/fangorn');
```

Comment:
Comment this out as well if we're going to disable these tests (so webpack doesn't have to parse this requirement).